During a nixpkgs-review on x86-64-Linux:

```sh
       > FAILED plotly/tests/test_optional/test_px/test_px_input.py::test_build_df_from_vaex_and_polars[vaex] - ModuleNotFoundError: No module named 'vaex'
       > FAILED plotly/tests/test_optional/test_px/test_px_input.py::test_build_df_from_vaex_and_polars[polars] - ModuleNotFoundError: No module named 'polars'
       > FAILED plotly/tests/test_optional/test_px/test_px_input.py::test_build_df_with_hover_data_from_vaex_and_polars[hover_data0-vaex] - ModuleNotFoundError: No module named 'vaex'
       > FAILED plotly/tests/test_optional/test_px/test_px_input.py::test_build_df_with_hover_data_from_vaex_and_polars[hover_data0-polars] - ModuleNotFoundError: No module named 'polars'
       > FAILED plotly/tests/test_optional/test_px/test_px_input.py::test_build_df_with_hover_data_from_vaex_and_polars[hover_data1-vaex] - ModuleNotFoundError: No module named 'vaex'
       > FAILED plotly/tests/test_optional/test_px/test_px_input.py::test_build_df_with_hover_data_from_vaex_and_polars[hover_data1-polars] - ModuleNotFoundError: No module named 'polars'
```

--- a/pkgs/development/python-modules/plotly/default.nix
+++ b/pkgs/development/python-modules/plotly/default.nix
@@ -120,6 +120,9 @@ buildPythonPackage rec {
     "test_dependencies_not_imported"
     # FAILED test_init/test_lazy_imports.py::test_lazy_imports - AssertionError: assert 'plotly' not in {'IPython': <module 'IPython' from '...
     "test_lazy_imports"
+    # requires vaex and polars, vaex is not packaged
+    "test_build_df_from_vaex_and_polars"
+    "test_build_df_with_hover_data_from_vaex_and_polars"
   ];
 
   pythonImportsCheck = [ "plotly" ];

